server/src/organizations/entities/organization.entity.ts

```diff
@@ -47,6 +47,6 @@ export class Organization {
   )
   transactions: Transaction[];
 
-  @Column({ type: 'text', unique: true })
+  @Column({ type: 'text', nullable: true })
```

Comment:
removed unique true because it was causing errors with seeding (because we use the same hardcoded img for multiple orgs)

added nullable true because we shouldn't require a profile img